django/test/utils.py

```diff
@@ -176,11 +177,23 @@ def setup_databases(verbosity, interactive, *, time_keeper=None, keepdb=False, d
             if first_alias is None:
                 first_alias = alias
                 with time_keeper.timed("  Creating '%s'" % alias):
+                    try:
+                        serialize_alias = connection.settings_dict['TEST']['SERIALIZE']
+                    except KeyError:
+                        serialize_alias = serialize_aliases is None or alias in serialize_aliases
+                    else:
+                        warnings.warn(
+                            'The `SERIALIZE` test database setting is deprecated '
+                            'as it can be inferred from the `databases` of the discovered '
+                            '`TransactionTestCase` that enable the `serialized_rollback` '
+                            'feature. It will be completely ignored from Django 5.0.',
```

Comment:
```suggestion
                            'The SERIALIZE test database setting is '
                            'deprecated as it can be inferred from the '
                            'TransactionTestCase.databases that enable the '
                            'serialized_rollback feature.', 
```